Makefile

```diff
@@ -379,7 +385,7 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 		-e 's|/@?main=$*.js|$*.js|' \
 		-e '/default\.js/d' \
 		-e 's|\.\./utils/watchwatchers.js|lib/watchwatchers.js|' \
-		-e '/$*.js/i\    <script src="lib/ngeo.js"></script>' $< > $@
+		-e '/$*.js/i\'$$'\n''    <script src="lib/ngeo.js"></script>'$$'\n' $< > $@
```

Comment:
`-e '/$*.js/i\'$$'\n    <script src="lib/ngeo.js"></script>'$$'\n' $< > $@`
no, not actually...